docs/apple-privacy-manifest.md

```diff
@@ -0,0 +1,431 @@
+1. In Xcode use the File->New->File menu to open the `Choose a new template for your file:` dialog box.
+1. Scroll down until you find the `App Privacy` template.
+1. Select the `App Privacy` template and then click `Next`
+1. In the `Save As` dialog, leave the filename as `PrivacyInfo` as that is the required name for the file.
+1. Click `Create` and close Xcode.
+1. Use Finder to copy the `PrivacyInfo.xcprivacy` file from the Xcode project to your documents folder for now.
+1. The Xcode project is no longer needed and can be deleted.
+
+You should now have a file named `PrivacyInfo.xcprivacy` under your documents folder with contents similar to:
+
+```xml
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<!--
+... omitted for brevity
+-->
+<plist version="1.0">
+<dict/>
+</plist>
+```
+Use your favorite text editor, like [Visual Studio Code](https://aka.ms/vscode), to open the file for editing.
+
+You can add the entries for the API's usage to the `PrivacyInfo.xcproject` as follows: 
+
+1. Edit the `PrivacyInfo.xcprivacy` file to appear as follows:
+    ```xml
+    <?xml version="1.0" encoding="UTF-8"?>
+    <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+    <!--
+    ... omitted for brevity
+    -->
+    <plist version="1.0">
+    <dict>
+        <key>NSPrivacyAccessedAPITypes</key>
+        <array>
+        </array>
+    </dict>
+    </plist>
+    ```
+    This adds the `NSPrivacyAccessAPITypes` key where each category usage will be added.
+1. Since the .NET runtime and BCL include API's from the [File timestamp][FileTimestampAPIs], [System boot time][SystemBootTimeAPIs], [Disk space][DiskSpaceAPIs] API categories, add the following to the `NSPrivacyAccessedAPITypes` array:
+    ```xml
+    <dict>
+        <key>NSPrivacyAccessedAPIType</key>
+        <string>NSPrivacyAccessedAPICategoryFileTimestamp</string>
+        <key>NSPrivacyAccessedAPITypeReasons</key>
+        <array>
+            <string>C617.1</string>
+        </array>
+    </dict>
+    <dict>
+        <key>NSPrivacyAccessedAPIType</key>
+        <string>NSPrivacyAccessedAPICategorySystemBootTime</string>
+        <key>NSPrivacyAccessedAPITypeReasons</key>
+        <array>
+            <string>35F9.1</string>
+        </array>
+    </dict>
+    <dict>
+        <key>NSPrivacyAccessedAPIType</key>
+        <string>NSPrivacyAccessedAPICategoryDiskSpace</string>
+        <key>NSPrivacyAccessedAPITypeReasons</key>
+        <array>
+            <string>E174.1</string>
+        </array>
+    </dict>
+    ```
+1. For the [NSProcessInfo.SystemUptime](https://learn.microsoft.com/en-us/dotnet/api/foundation.nsprocessinfo.systemuptime), a reason code of `35F9.1` is needed since that is the only reason code available. But since the .NET runtime and BCL already included that category and reason, there is nothing additional to add.
```

Comment:
```suggestion
1. For the [NSProcessInfo.SystemUptime](https://learn.microsoft.com/dotnet/api/foundation.nsprocessinfo.systemuptime), a reason code of `35F9.1` is needed since that is the only reason code available. But since the .NET runtime and BCL already included that category and reason, there is nothing additional to add.
```

Remove locale id

Reply:
Fixed